explorer/client/src/__tests__/e2e.test.ts

```diff
@@ -391,6 +398,9 @@ describe('End-to-end Tests', () => {
         it('to go to the last page', async () => {
             const address = 'ownsAllAddress';
             await page.goto(`${BASE_URL}/addresses/${address}`);
+            const btn1 = await page.$('#groupCollection > div:nth-child(1)');
```

Comment:
we re-use `'#groupCollection > div:nth-child(1)'` selector several times, could we make it a `const` variable with a name describing what the selector is for ?

Reply:
A refactoring of the test suite code is a good idea and certainly something to consider post-Devnet.

Reply:
my concern is more for our own future understanding of this code: it's good to give descriptive names to magic constant values in the codebase.